app/retail/views.py

```diff
@@ -994,6 +994,27 @@ def activity(request):
 
     return TemplateResponse(request, 'activity.html', context)
 
+@ratelimit(key='ip', rate='30/m', method=ratelimit.UNSAFE, block=True)
+def create_status_update(request):
+    response = {}
+    if request.POST:
+        profile = request.user.profile
+        kwargs = {
+            'activity_type': 'status_update',
+            'metadata': {
+                'title': request.POST.get('data')
+            }
+        }
+        kwargs['profile'] = profile
+        try:
+            Activity.objects.create(**kwargs)
+            response['status'] = 200
+            response['message'] = 'Status updated!'
+        except Exception as e:
```

Comment:
What exceptions are we planning on handling here?

Reply:
Come in handy for catching and logging errors

Reply:
Yeah but is there a specific exception we are planning on seeing? We could put try: catch blocks around everything but catching exceptions costs performance, so we need to use it only in cases where we need it.

Reply:
Should I remove it?